src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -348,6 +349,10 @@ public function askQuestion(Question $question)
 
         if (!$this->questionHelper) {
             $this->questionHelper = new SymfonyQuestionHelper();
+
+            if ($this->inputStream) {
```

Comment:
It should be possible to set after creating the SymfonyQuestionHelper instance.
What about detecting in `setInputStream()` if `$this->questionHelper` is not null and then calling `setInputStream()` on the SymfonyQuestionHelper object?